src/WebJobs.Script.WebHost/Management/GetTriggersResult.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using static Microsoft.Azure.WebJobs.Script.WebHost.Management.FunctionsSyncManager;
```

Comment:
can remove this using if you make the changes suggested below

Reply:
removed it.